src/snapred/backend/data/LocalDataService.py

```diff
@@ -142,7 +142,7 @@ def _readInstrumentParameters(self) -> Dict[str, Any]:
             raise _createFileNotFoundError("Instrument configuration file", self.instrumentConfigPath) from e
 
     def readStateConfig(self, runId: str, useLiteMode: bool) -> StateConfig:
-        diffCalibration = self.calibrationIndexer(runId, useLiteMode).readParameters()
+        diffCalibration = self.calibrationIndexer(runId, useLiteMode).readParameters(VersionState.LATEST)
```

Comment:
This will return the `current` index.  Should it?

Reply:
It should not, this needs to be fixed.  I will be removing the LATEST -> current flattening, as that isnt correct and should never be used.  i.e. loading should always require a version, saving would never refer to 'current'